docs/reference/modules/discovery/quorums.asciidoc

```diff
@@ -0,0 +1,181 @@
+[[modules-discovery-quorums]]
+=== Quorum-based decision making
+
+Electing a master node and changing the cluster state are the two fundamental
+tasks that master-eligible nodes must work together to perform. It is important
+that these activities work robustly even if some nodes have failed, and
+Elasticsearch achieves this robustness by only considering each action to have
+succeeded on receipt of responses from a _quorum_, a subset of the
```

Comment:
```suggestion
succeeded only on receipt of responses from a _quorum_, which is a subset of the
```

Reply:
I removed the suggested change `succeeded on` -> `succeeded based on` but accepted the other one. "Based on" to me suggests there might be additional criteria for success, whereas here "receipt from a quorum" is the only requirement.